docs/src/components/Header/Header.js

```diff
@@ -15,8 +15,18 @@ const Header = ({ branch, cascaraVersion }) => {
       post={
         <div>
           <Flex>
-            <Tag content={cascaraVersion} />
-            {branch !== 'main' && <Tag content={branch} />}
+            {cascaraVersion && (
+              <a
+                href={`https://github.com/Espressive/cascara/packages/361656?version=${cascaraVersion}`}
+              >
+                <Tag content={cascaraVersion} />
+              </a>
+            )}
+            {branch && branch !== 'main' && (
+              <a href={`https://github.com/Espressive/cascara/tree/${branch}`}>
+                <Tag content={branch} />
+              </a>
+            )}
```

Comment:
Adds links from the header to the actual release page or branch in Github 💅🏽

Reply:
nice